### ISSUE TITLE
Skull and Bones (2853730)

### ISSUE THREAD

**Issue by Lifeismana** [ValveSoftware/Proton#8018]
# Compatibility Report
- Name of the game with compatibility issues: Skull and Bones
- Steam AppID of the game: 2853730

## System Information
- GPU: RTX 2080
- Video driver version: nvidia-open 560.35.03
- Kernel version: 6.10.6-zen1-1.1-zen 
- Link to full system information report as Gist: https://gist.github.com/Lifeismana/c8a8fac53210a22b6eb8760a28bfb058
- Proton version: Proton Experimental

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Game doesn't launch/crashes after showing a black window for 2 seconds

[steam-2853730.log](https://github.com/user-attachments/files/16732765/steam-2853730.log)
(fwiw steam still shows the game as running after it crashes but i guess it's the uplay launcher)
## Reproduction

- Launch the game
- Log into your ubisoft account (if needed) then let the Uplay launcher do its things
- The game's splashscreen pops-up followed soon after by a game windows before closing by itself

**Comment by Lifeismana** [comment]
Seeing that there a [protondb report](https://www.protondb.com/app/2853730) that it runs fine on an amd gpu makes me think that it's an nvidia specific problem (which would explain why it runs fine on the steam deck)

Fwiw, disabling explicit sync doesn't help with that crash

I saw a [post on nvidia's forum](https://forums.developer.nvidia.com/t/farcry5-nvidia-555-560-no-longer-works/303879) about Far Cry 6 not working anymore on recent nvidia drivers but since skull&bones isn't using the same engine i kinda doubt it's linked, although it still made me want to try older drivers.
Sadly i was unable to install an older working version (either breaks kde or doesn't install)

**Comment by Shatrico** [comment]
I tried to start the for months and around 1-2 months i was able to get it to start with Proton but got kicked by BattleEye until the Release day on Steam where I was able play the game. So i think there could be some issues with Nvidia again.

System Specs:
CPU: AMD Ryzen 9 3950X
RAM: 64 GB DDR4
GPU: AMD AMD Radeon RX 7900 XT
OS: EndeavourOS 
Kernel: 6.10.6-zen1-1-zen (but also working with the non zen version)

**Comment by Tiagoquix** [comment]
Does it help if you launch the game with the following launch options?
```
PROTON_DISABLE_NVAPI=0 PROTON_HIDE_NVIDIA_GPU=0 DXVK_CONFIG="dxgi.hideNvidiaGpu = False" %command%
```
(Also try with values `1`, `1` and `= True`, respectively).

**Comment by Lifeismana** [comment]
Nah, sadly it doesn't help, tried both cases (1 1 True & 0 0 False) but it still crashes

**Comment by Lifeismana** [comment]
Gave Proton 8.0-5 a try and oddly enough vs experimental, it shows the intro for a few seconds but then subsequently also crashes

**Comment by Thermatix** [comment]
So the game works flawlessly now for the most part (whatever issue with the anti-cheat they had is now not an issue any more).

BUT there is one problem

Sometimes the mouse cursor doesn't get captured properly and I have two monitors and the mouse just escapes the game window when I'm turning the in-game camera clock-wise.

This used to be resolvable by just restarting the game and making sure not to select anything else when the game is starting.

Now though, It won't capture consistently, like at all.

I'm not sure if this is a Proton issue but I'm fairly certain it's not the game itself, because  If i select the "lock mouse cursor" option in game it seems to work when the game first boots but then stops when it gets to the loading screen. Also This seems to only be an issue with Skull & Bones as I don't have this issue with DarkTide.

The other thing I've noticed is that when the mouse-cursor has failed to be captured, when you alt tab to another program (like fire-fox) the mouse cursor still looks like the in-game one and clicking any-where forces the game to the front, as if you're still in the game window.

This seems to be an issue with ProtonGE-10.4 and Proton-10 though the Second behavior is slightly different, in ProtonGE, the clicking any where snaps the game back into focus, Proton-10 does not; also if I select the second screen then try to move back  with Proton-10 the mouse cursor vanishes, with ProtonGE-10 it does not.

This seems to have been fixed with Proton Experimental BUT there is a regression, where the game screen is rendered to one quarter the screen size in to the top left with the rest of the screen being black.

Proton Next just crashes on Launching Ubisoft connect

Proton Hotfix Has the same issue with mouse capturing as Proton-10 & ProtonGE-10.
